blobs/quickstarts/JavaScript/V12/nodejs/index.js

```diff
@@ -0,0 +1,89 @@
+const { BlobServiceClient } = require("@azure/storage-blob");
+const { v1: uuidv1 } = require("uuid");
+require('dotenv').config()
+
+const AZURE_STORAGE_CONNECTION_STRING =
+  process.env.AZURE_STORAGE_CONNECTION_STRING;
+
+if (!AZURE_STORAGE_CONNECTION_STRING) {
+  throw Error("Azure Storage Connection string not found");
+}
+
+async function main() {
+  console.log("Azure Blob storage v12 - JavaScript quickstart sample");
+  // Quick start code goes here

```

Comment:
should we remove this comment - looks like it's from a template?

Reply:
The `// Quick start code goes here` is meant to indicate where the reader of the quickstart is to copy each code section in. I can go either way with keep it or remove it.

Reply:
@tamram Hi Tamra, I made the change, can you please merge?